spec.md

```diff
@@ -99,7 +97,8 @@ The `Span` interface must have the following capabilities:
 
 The `Tracer` interface must have the following capabilities:
 
-- Start a `Span` that has no parent, commonly referred to as a *root* `Span` **(py: `start_trace`, go: `StartTrace`)**
+- Start a new child `Span` with a given operation name. With no parent, commonly referred to as a *root* `Span`, no trace attributes exist. With a parent any trace attributes must be passed through into the created child. **(py: `create_span`, go: `CreateSpan`)**
```

Comment:
If I understand your last proposal right, it fails to allow a case where traces are started at "edge" `create_span` calls, but not started at "interior" calls, because it is a global property of the tracer and the two categories are not distinguished.  So it still requires instrumentors to do if-guards in all their subsequent instrumentation logic.  Having a separate `start_trace` method will allow this distinction to exist without boilerplate guards on every subsequent `create_span` call (which would also hide throughput counts from any interested tracer implementations, as I noted above).